regtests/client/python/cli/polaris_cli.py

```diff
@@ -55,6 +67,27 @@ def execute(args=None):
                 PolarisCli._try_print_exception(e)
                 sys.exit(1)
 
+    @staticmethod
+    def _merge_with_config(options):
+        config = PolarisCli._load_config()
+        if not config:
+            return
+
+        for attr in ['client_id', 'client_secret', 'host', 'port']:
+            if not getattr(options, attr) and config.get(attr):
+                setattr(options, attr, config[attr])
+
+    @staticmethod
+    def _load_config():
+        try:
+            with open(POLARIS_CLI_JSON, "r") as file:
```

Comment:
Only support the current location, need to support the home dir(`~`) as well.

Reply:
probably want to allow a cli arg like -c configFile as well